deeppavlov/configs/kbqa/kbqa_eng.json

```
@@ -0,0 +1,282 @@
+{
+  "dataset_reader": {
+    "class_name": "kbqa_reader",
+    "data_path": "{DOWNLOADS_PATH}/wikidata_eng/"
+  },
+  "dataset_iterator": {
+    "class_name": "data_learning_iterator"
+  },
+  "chainer": {
+    "in": ["x"],
+    "in_y": ["y"],
+    "pipe": [
+      {
+        "id": "classes_vocab",
+        "class_name": "simple_vocab",
+        "fit_on": [
+          "y"
+        ],
+        "save_path": "{MODEL_PATH}/relation_prediction/classes.dict",
+        "load_path": "{MODEL_PATH}/relation_prediction/classes.dict",
+        "in": "y",
+        "out": "y_ids"
+      },
+      {
+        "in": [
+          "x"
+        ],
+        "out": [
+          "x_prep"
+        ],
+        "class_name": "dirty_comments_preprocessor"
+      },
+      {
+        "in": "x_prep",
+        "out": "x_tok",
+        "id": "my_tokenizer",
+        "class_name": "nltk_tokenizer",
+        "tokenizer": "wordpunct_tokenize"
+      },
+      {
+        "in": "x_tok",
+        "out": "x_emb",
+        "id": "my_embedder",
+        "class_name": "fasttext",
+        "load_path": "{DOWNLOADS_PATH}/embeddings/wordpunct_tok_reddit_comments_2017_11_300.bin",
+        "pad_zero": true
+      },
+      {
+        "in": "y_ids",
+        "out": "y_onehot",
```

```suggestion
        "out": ["y_onehot"],
```
everywhere for `"in"` and `"out"`

---

Changed.